### DISCUSSION TITLE
svc : The term 'svc' is not recognized as the name of a cmdlet, function, script file, or operable program.

### DISCUSSION THREAD

**Discussion by abysshtml** [voicepaw/so-vits-svc-fork#706] [Q&A] [answered]
I don't know much in particular about programming, is someone able to help me find out how I can get this to work?
![image](https://github.com/voicepaw/so-vits-svc-fork/assets/63281580/9890cc9e-8e40-40b7-a797-5626f5fe1845)

**Comment by Grownz** [comment, marked as answer]
You're probably in the wrong environment.
try `conda activate <putyourenvnamehere>` first.
